lib/TH/THMemoryFile.c

```diff
@@ -479,7 +479,7 @@ static size_t THMemoryFile_writeLong(THFile *self, long *data, size_t n)
     size_t i;
     for(i = 0; i < n; i++)
     {
-      ssize_t nByteWritten;
+      size_t nByteWritten;
```

Comment:
changing the ssize_t to size_t is likely breaking the contbuilds. See https://travis-ci.org/torch/torch7/builds/134239495